### PR TITLE
db connection and dao files are added

### PR DESCRIPTION
**Pull request by jitendra2329**
I have added Dao and Connection file in respective packages.

### CODE REVIEW

**Comment by Bhavya-verma-knoldus** [comment]
This PR also has target files which should not be there.

File: app/controllers/Deliveries.scala [hunk 1]
```diff
@@ -1,22 +1,41 @@
...
+  controllerComponents: ControllerComponents,
+  postgresDAO: PostgresConnection
+) extends   AbstractController(controllerComponents)  with DeliveriesController {
   override def getById(request: Request[AnyContent], merchantId: String, id: String): Future[GetById] = {
     Future.successful(
       deliveriesService.getByid(merchantId, id) match {
         case Left(_) => GetById.HTTP404
-        case Right(delivery) => GetById.HTTP200(delivery)
+        case Right(delivery) =>
+          index
```
Comment (Bhavya-verma-knoldus) [inline app/controllers/Deliveries.scala]: Seems like this was used for testing purpose and should be removed now.

File: app/database/PostgresConnection.scala [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import com.nashtech.delivery.v1.models.{Address, Contact, Delivery}
+import play.api.db._
+import play.api.libs.json._
+
+import javax.inject.Inject
+
+class PostgresConnection @Inject()(db: Database) {
+  def getAll(sqlForD: String): Seq[Delivery] = {
+    db.withConnection { implicit connection =>
+      SQL(sqlForD).as(deliveryParser.*)
```
Comment (Bhavya-verma-knoldus) [inline app/database/PostgresConnection.scala]: Have you tried using Delivery.parser from the generated parser?

File: build.sbt [hunk 3]
```diff
@@ -7,20 +7,48 @@ lazy val root = (project in file("."))
...
+ "joda-time" % "joda-time" % "2.12.5",
+  "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.14.2",
+  "org.postgresql" % "postgresql" % "42.6.0",
 )
+libraryDependencies ++= Seq(
+  "mysql" % "mysql-connector-java" % "5.1.41"
+)
+
+
+//libraryDependencies += "com.typesafe.play" %% "play-slick" % "5.2.0-M2"
```
Comment (Bhavya-verma-knoldus) [inline build.sbt]: If these dependencies are not needed, let's remove these.

File: app/database/PostgresConnection.scala (hunk 2 above)
Comment (jitendra2329) [reply app/database/PostgresConnection.scala]: No, I was not aware of it. I will check.

File: build.sbt (hunk 3 above)
Comment (jitendra2329) [reply build.sbt]: yes sure, will do the same.

File: app/controllers/Deliveries.scala (hunk 1 above)
Comment (jitendra2329) [reply app/controllers/Deliveries.scala]: Yes, will remove it.

**Comment by Bhavya-verma-knoldus** [comment]
Kindly make the requested changes

**Pull request merged**